scripts/windows_install_mingw64.ps1

```diff
@@ -0,0 +1,33 @@
+$ErrorActionPreference = "Stop"
+
+echo "starting mingw64 installation"
+
+Invoke-WebRequest -UseBasicParsing -uri "https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe" -OutFile msys2.exe ; 
+if (!$?) { throw 'cmdfail' }
+
+.\msys2.exe -y -oC:\ ;
+Remove-Item msys2.exe ;
+
+
+function msys() { C:\msys64\usr\bin\bash.exe @('-lc') + @Args; } ; 
```

Comment:
I think this needs error handling? Otherwise one of the steps could fail, and we'd  treat it as successful nonetheless?

Reply:
added the line `if (!$?) { throw 'cmdfail' }` after this line. would it be enough for error handling?

Reply:
Yep, that should be enough.